package.json

```diff
@@ -41,6 +41,7 @@
 		"@tanstack/query-core": "^4.36.1",
 		"@tanstack/query-persist-client-core": "^4.36.1",
 		"@tanstack/query-sync-storage-persister": "^4.36.1",
-		"@tanstack/svelte-query": "^4.36.1"
+		"@tanstack/svelte-query": "^4.36.1",
+		"datetrigger": "github:madkarmaa/DateTrigger"
```

Comment:
please use npm so we have stability, audibility, etc

Reply:
I'll have to publish a version, I'll do that

Reply:
done in 890fc7e